spec/controllers/user/swaps_controller_spec.rb

```diff
@@ -329,8 +327,57 @@
             email: "")
     end
 
-    let(:mobile_phone) do
-      build(:mobile_phone, user_id: 122, number: "07400 123456", verified: true)
+    let(:swap_user) do
```

Comment:
GH has chosen a weird diff here - the "when users don't have an email" test is basically the same, the "when users don't have an mobile number" test is new.